#!/usr/bin/python

import traceback

try:
	import magic_square
except:
	print(traceback.format_exc())
